IntegrationTestSuite/WebservicesIntegrationTests/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DotNetZip" version="1.10.1" targetFramework="net452" />
+  <package id="DotNetZip" version="1.13.7" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net452" />
-  <package id="NLog" version="4.5.2" targetFramework="net452" />
-  <package id="NUnit" version="3.10.1" targetFramework="net452" />
-  <package id="NUnit3TestAdapter" version="3.10.0" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net452" />
+  <package id="NLog" version="4.7.0" targetFramework="net452" />
+  <package id="NUnit" version="3.12.0" targetFramework="net452" />
+  <package id="NUnit3TestAdapter" version="3.16.1" targetFramework="net452" developmentDependency="true" />

```

Comment:
Was this change intentional?
What does it mean?

Reply:
This is, because I updated nuget packages in my solution.

Reply:
I saw that, but why is the developmentDependency="true" added?

Reply:
changed